### ISSUE TITLE
Change kaibokan's hullcode from DDE to PF

### ISSUE THREAD

**Issue by LostLogia4** [KC3Kai/KC3Kai#2019]
Kaibokans (coastal defense ships) lacks torpedoes, among other things, making them unsuitable to perform destroyer's tasks. Loosely they could do some of DD's tasks, but they're not quite the same thing. Either way, we should change away from that to avoid confusion.

http://kancolle.wikia.com/wiki/Thread:543852#165

**Comment by nonsensed** [comment]
Eh, I think I'm going to stop looking into this. In short, the DE or PF designation isn't going to matter much and it's more important to be consistent across the wiki(a) and viewers. If other sources for kancolle are also ok with PF, then it's ok for me as well. But it's not worth pushing PF for kc3 if other sources don't also use it.

(will be "fun" if we have to remap old quests on whether these new ships are allowed for them or not)

**Comment by c933103** [comment]
over wikia the designation is still undecided although some opinion favor the use of the code PF. and last time I checked kancollewiki.net they still haven't give it any short form while they use destroyer escort for the type translation. 74EO and poi browser currently use DDE. English KCVs mostly use DE.

**Comment by nonsensed** [comment]
per discord, via @Tsubakura 

source that says Shimushu would be designated "PF"
http://archive.hnsa.org/doc/id/oni222j-japan-navy/pg107.htm

I looked around on that site. I guess there was a later revision designated DE-UN-1.
http://archive.hnsa.org/doc/id/oni222j-japan-navy/pg111.htm

Anyway, the change has been made. Kc3kai is calling them "PF". I'm glad to have this over with. Hopefully.

**Comment by c933103** [comment]
https://github.com/poooi/poi/issues/1448

**Comment by nonsensed** [comment]
I'm ambivalent toward all sides of the argument.

I'm reopening this giantmonsterwontfreakingdie issue. If anyone else involved in kc3 wants to voice their opinions, please do so.

Above all else, my opinion is that as many sources as possible should support a single ship abbreviation to avoid player confusion. This is the most important thing. Of secondary importance is picking an abbrevation which matches with previous naming convention standards, which heavily leans toward using an official USN abbreviation. There is a lot more to say, but I will leave it at that. (refer to the first sentence in this post)

**Comment by ethrundr** [comment]
Taking a short visit to GitHub and found this discussion opened. So, here's my opinion.

Regarding "Coastal Defense Ships", there are two terms that could be closely related.
In IJN itself, Coastal Defense Ships are aligned with Patrol Boats, which by then comes the Hull code: PF.

However, if we look at the documentation of these ships, especially one made by Allied side, these ships are called as "Escort ships". Thus, the code DE.

If we would like to regard to the Wikipedia, it is clearly stated in the DE description page that "Kaibokan of IJN are made to fulfill escort role.". Also, there are lot of other documentations and descriptions that categorized Kaibokans as Escorts, not Frigates. As reference, PF stands for "Patrol Frigates/Boats", which Frigates itself just receiving classification post World War II (around at 1975).

 Also, since Kaibokan is by no means a Destroyer, thus the usage of DDE should be dropped: DDE is for a Destroyer that functioned as Escort ships. DE has less tonnage and capacity compared to DDE.

There's my 2 cents regarding these issues. Hopefully can be helpful.

**Comment by Tsubakura** [comment]
Mhm, thanks for the reply. The thing is though, Escort does **not** mean DE, Escort itself is a category of ships that fulfilled escort duties. In other words, PF, DE and even the Kaiboukan all fall under the Escort ship. 

Anyway, I have compiled the arguments from all sides. Remember that the goal here is not to only provide a standard for everyone to use, but also to use a classification that is as historically accurate as possible, while trying to avoid issues in the future. I have involved @nonsensed and @dragonjet into our group DM to settle this matter, but I want you guys feed on this as well.

**Arguments that the Kaiboukan should be DE (Destroyer Escort):**
- The community has already accepted it as the correct classification, without even bothering to look through the historical facts.
- They have the same role generally during the WW2, even though they lack all the armanents and specifications of an American Destroyer Escort.
- The US Army identification magazine used both DE and PF interchangeably: https://archive.org/details/USANJOR194405-nsia (Note: Magazine, thus most likely not official practice)
- The documents could all have been written by someone from the Royal Navy or they could've exchanged info with them, which does not prove that the USN themselves uses the classification PF themselves.
- Nobody cares about the Naval Treaties, so we shouldn't either.

**Arguments that the Kaiboukan should be PF (Patrol Frigate):**
- The ONI has classified the Shimushu-class as PF: http://archive.hnsa.org/doc/id/oni222j-japan-navy/pg107.htm
- Page 40 of the USN Technical Mission to Japan classified the Kaiboukan as PF: http://www.fischer-tropsch.org/primary_documents/gvt_reports/USNAVY/USNTMJ%20Reports/USNTMJ-200H-0602-0659%20Report%20S-03.pdf
- The IJN does not treat any ships with no torpedo capabilities as destroyers.
- The Republic of China's Navy classified these ships as PF when they obtained them post-war. Likewise, the American Destroyer Escorts were given a different classification, which means there are distinctions.
- In-game, they cannot effectively replace DDs in most of the expeditions and map branching rules. It is evidence enough that they are not treated as destroyers, as genuine destroyer escorts should be able to effectively replace them.
- DE are subject to the London Naval Treaty of 1930 if they have a displacement of over 2000 ton, any gun above 155cm, more than 4 guns above 76mm, the ability to launch torpedoes and the speed of 20 knots or above. The Shimushu however, is nowhere near 2000 ton, does not have any gun above 155cm, only has 3 guns above 76mm at most, no torpedo capabilities and has a speed of 19.7 knots. They are not a destroyer by London Naval Treaty definitions.
- The Kaiboukan were built with the purpose of settling fishing disputes between USSR and Japan, not for use as an escort in war.
- The Kaiboukan lacks the armanent to be as combat effective as the American Destroyer Escorts. While the lightweight part does fit the role of the USN DE, their capabilities dont.
- The US Army identification magazine used both DE and PF interchangeably: https://archive.org/details/USANJOR194405-nsia (Note: Magazine, thus most likely not official practice)


**Arguments that the Kaiboukan should be EE (Escort):**
- The in-game shipcards themselves literally state that they're Escorts. Not PF or DE, but Kaiboukan and Escort.
- NHHC has provided a statement that they are Escorts. They have also provided the following trusted source: http://www.combinedfleet.com/Kaibokan.htm
- Escort can basically mean PF, DE and Kaiboukan, pretty much resolving everything in the debate.
- Although the ONI has assigned the first 3 Kaiboukan classes as PF, the 4th one is actually assigned as DE: http://archive.hnsa.org/doc/id/oni222j-japan-navy/pg115.htm
- Nobody can claim that EE is wrong, because it literally means Escort. In case foreign escorts gets added, there will be no problems at all if it gets the same API as the Kaiboukan.


**Arguments that the Kaiboukan should be KB (Kaiboukan):**
- Is there even an explanation needed? The ships are given the tag 海防 in their cards, so why don't we just classify them KB? 

I have personally drawn a conclusion from all the info that we have gathered from the historians, so I let you guys decide on this matter.

**Comment by nonsensed** [comment]
@KC3Kai/translators 

After a lot of discussion, we finally agreed that, due to the present circumstances, there will be a vote soon on what ship abbreviation to assign to the Kaikoban API ID. I've agreed to use the results of this for the `EN` ship abbreviation.

There is currently open discussion on the matter and then there will be a vote eventually. The first discussion is on whether `DE` should be used, or if we should use a different abbreviation-- `EE`, `PF` or `KB`. You can view some discussion in various places, but here are two places to look:
https://www.reddit.com/r/kancolle/comments/6b34rr/discussion_abbreviation_to_use_for_kaiboukan/
http://kancolle.wikia.com/wiki/Thread:558979

If anyone would like to throw their opinion in, the reddit topic will likely be seen by the most people. When it is time to vote, I will try to link to it as soon as possible.

If any translator for other languages does not like the result of this poll, I don't want to force you to use it. I will be using it for `EN` translations, however.

For translation purposes, I will be handling things like this:
Broad ship type: (results of this poll)
Ship type: Kaikouban
Ship class: Shimushu-class, Etorofu-class, etc.

Broad ship type is important because it is used in things like quest translations.

**Comment by ethrundr** [comment]
Taking into note @Tsubakura's explanation, I think the appropriate abbreviation then will be `EE`, as it'll globalized Escort ships.

But one problem arise from here: 
- When the actual DE ships like Matsu-class and Tachibana-class got implemented, which one will they use: DE or EE? Since from explanation above, EE does include DE inside.
This, I think, need to be resolved first before any abbreviation is decided.

_ps: For anyone has access to reddit, please help me to take note on that there. Indonesian internet connection is bit suck if we're going to access reddit (it's blocked by the government, ugh)._

**Comment by nonsensed** [comment]
Hmm, the formatting is ugly, but
https://pastebin.com/FFG8EbFb

TL;DR - many people have many different opinions on this subject. `DE` and `PR` have been popular in this discussion. We will only be deciding whether to use `DE` or not for now. We will only move on to the other options if the voting decides not to use `DE`. The vote has not started yet.

**Comment by nonsensed** [comment]
@KC3Kai/translators 

> As it's been 2 days since the debate which can be seen at
> http://kancolle.wikia.com/wiki/Thread:558979
> and
https://www.reddit.com/r/kancolle/comments/6b34rr/discussion_abbreviation_to_use_for_kaiboukan/
(note: the pastebin above contains almost all of the reddit discussion)
> It is time that we settle on whether we want to use DE or not.
> 
> Keep in mind the following regarding this poll:
> 
- This poll will close on 20-05-2017 at 05:00 JST. It is the same time when the quest resets take place.
- You must be logged in with a gmail.
- You will not be able to see the results. The results will be announced when the poll closes.
> 
> You are recommended to read through the discussions first to gain an understanding as to what is happening, before you decide to cast your vote.
> 
> You can access the poll through this link.
https://docs.google.com/forms/d/e/1FAIpQLSemJNe3MqvLqWS0YcPW1aqgSkzMqpofFiDydgvEaKLIBQwLSQ/viewform?c=0&w=1
> 
> Good luck.

I will say it again:

I plan to use this abbreviation for the `EN` translation in kc3kai. I don't plan to force any other language to use this if they don't want to.

By all means, please vote. If `DE` does not win, we will vote on `EE`, `PF`, or `KB` after the poll closes. If you would like to share this vote with a community, I ask that you please include the discussion links above if at all possible. The more people know about the subject, the better a decision people can make.

**Comment by nonsensed** [comment]
> Hi everyone,
> 
> With regards to the poll on the abbreviation to use for the newly released IJN Kaiboukan, the voting result is as follow:
> 
> DE: 386 votes Something else: 388
> 
> The results were debated internally on how we should interpret this and move forward. We initially believed that we would have a clear winner with a decent margin but on hindsight, we did not expect this result - a virtual tie.
> 
> 2 proposals were made: the first was to run a second poll but to include DE (DE, PF, KB, EE). It was obviously going to be a formality because any sensible person can draw the conclusion with the result of the current poll. The other proposal was to declare DE the winner in view that despite DE losing by 2 votes, it is technically the majority, the winner, because the other 388 votes from "something else" is a combination of options that would have no chance to win against DE when pitted against it one-on-one.
> 
> The group are in view that the latter should be the answer and the former pointless, and that we have heard from you that you guys wanted to get this over and done. The exception is Tsubakura, representing Wikia, does not want to accept this resolution and have declared that Wikia will proceed with further polls on this matter. As such, any further discussions on Wikia shall be deemed as the sole decision of Wikia and in no way r/kancolle, our discord community, and other communities/platforms are in support of it, unless otherwise stated by themselves or their position changes.
> 
> With this, we would like to notify you guys that we have decided that we would accept DE as the majority vote and this debate closed.
> 
> Let us all return to our boatgirls and enjoy the game, as we should.

@KC3Kai/translators 

I will change the `EN` and inactive language files from `PF` to `DE`. I'll open an issue in the translation repo for this to be changed for other languages, but you do not have to if you don't want to.

**Comment by ethrundr** [comment]
@nonsensed as `ID` language active translators are now only me and @freezebear , please change `ID` as well if you have time to.

**Comment by mikenzb** [comment]
@ethrundr done. 😄
